README.md

```diff
@@ -14,25 +14,28 @@ It's starts a local TurboRepo server (on port `9080`) and uses Github artifacts
   uses: felixmosh/turborepo-gh-artifacts@v1
   with:
     repo-token: ${{ secrets.GITHUB_TOKEN }}
-    server-token: ${{ secrets.TURBO_SERVER_TOKEN }}
 ```
 2. Make turbo repo work with the local server
 ```yaml
 - name: Build
-  run: yarn build --api="http://127.0.0.1:9080" --token="${{ secrets.TURBO_SERVER_TOKEN }}" --team="foo"
+  run: yarn build
+  env:
+    TURBO_API: 'http://127.0.0.1:9080'
+    TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+    TURBO_TEAM: 'foo'
+    TURBO_REMOTE_ONLY: true
```

Comment:
Why do we need the REMOTE_ONLY flag?

Reply:
I added it because it is in the TurboRepo docs, [commented out in the build step of third example](https://turbo.build/repo/docs/ci/github-actions). 

```yaml
# To use Remote Caching, uncomment the next lines and follow the steps below.
env:
  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
  TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
  TURBO_REMOTE_ONLY: true
```

Though now that I look at it, it's only in the `npm` example. They don't have all the environmental variables documented (I think Vercel would much prefer us to use their cacheing service 😉).

So if you want to leave it out, I can definitely make that edit.

Reply:
Though I don't know what issues could happen if it is left out, so we might want to make a comment about it... I'll do some digging in the TurboRepo issues/PRs and see if I can find out more details about that flag.

Reply:
Found it in the [TurboRepo docs](https://turbo.build/repo/docs/reference/command-line-reference#--remote-only) lol, its the same as the cli flag `--remote-only`.

> Default `false`. Ignore the local filesystem cache for all tasks. Only allow reading and caching artifacts using the remote cache.

So we can point to the docs and users can make their own decisions about what flags/env variables they need.